.changeset/chatty-humans-collect.md

```diff
@@ -0,0 +1,7 @@
+---
+'@graphql-mesh/fusion-runtime': minor
+---
+
+Extract subgraph execution batching logic outside, so batching is handled by the Gateway not Stitching
+
+**BREAKING**; `UnifiedGraphHandlerOpts` no longer takes `batch` option, it is handled by the runtime itself
```

Comment:
this facing outside or just an internal breaking change? should we improve the changeset?

Reply:
Affecting only this package. What do you think we should improve?